src/coreclr/src/jit/optcse.cpp

```diff
@@ -2907,12 +2909,6 @@ class CSE_Heuristic
                 // Assign the ssa num for the ref use. Note it may be the reserved num.
                 ref->AsLclVarCommon()->SetSsaNum(cseSsaNum);
 
-                // If it has a zero-offset field seq, copy annotation to the ref
-                if (hasZeroMapAnnotation)
-                {
-                    m_pCompiler->fgAddFieldSeqForZeroOffset(ref, fldSeq);
-                }
```

Comment:
Deleting this?

Reply:
Yes, originally there were two places the zero map annotation got added -- in the def case, and below the join of the use/def. So it was possible to try and add the zero map annotation to the def twice.

Now we just rely on the common one below the join.